tutorials/intro/plot_introduction.py

```diff
@@ -1,404 +1,293 @@
-
-raw.save('sample_audvis_meg_raw.fif', tmin=0, tmax=150, picks='meg',
-         overwrite=True)
-
-##############################################################################
-# Define and read epochs
-# ^^^^^^^^^^^^^^^^^^^^^^
+###############################################################################
+# Preprocessing
+# ^^^^^^^^^^^^^
 #
-# First extract events:
-
-events = mne.find_events(raw, stim_channel='STI 014')
-print(events[:5])
-
-##############################################################################
-# Note that, by default, we use stim_channel='STI 014'. If you have a different
-# system (e.g., a newer system that uses channel 'STI101' by default), you can
-# use the following to set the default stim channel to use for finding events:
-
-mne.set_config('MNE_STIM_CHANNEL', 'STI101', set_env=True)
-
-##############################################################################
-# Events are stored as a 2D numpy array where the first column is the time
-# instant and the last one is the event number. It is therefore easy to
-# manipulate.
+# MNE-Python supports a variety of preprocessing approaches and techniques
+# (maxwell filtering, signal-space projection, independent components analysis,
+# filtering, downsampling, etc), see the full list of functions in the
+# :mod:`mne.preprocessing` and :mod:`mne.filter` submodules. Here we'll quickly
+# clean up our data by adding SSP projectors for eye movements and heartbeats;
+# we'll also plot the projectors to show the field characteristics that we're
+# projecting out:
+
+# eye movement/blink projectors
+kwargs = dict(n_grad=1, n_mag=1, n_eeg=1, average=True, no_proj=True)
+eog_projs, eog_events = mne.preprocessing.compute_proj_eog(raw, **kwargs)
+
+# heartbeat projectors
+kwargs.update(n_eeg=0, reject=None)
+ecg_projs, ecg_events = mne.preprocessing.compute_proj_ecg(raw, **kwargs)
+
+raw.add_proj(eog_projs + ecg_projs)
+mne.viz.plot_projs_topomap(eog_projs + ecg_projs, info=raw.info)
+
+###############################################################################
+# Detecting experimental events
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 #
-# Define epochs parameters:
-
-event_id = dict(aud_l=1, aud_r=2)  # event trigger and conditions
-tmin = -0.2  # start of each epoch (200ms before the trigger)
-tmax = 0.5  # end of each epoch (500ms after the trigger)
-
-##############################################################################
-# Exclude some channels (original bads + 2 more):
+# The sample dataset includes several "STIM" channels that recorded electrical
+# signals sent from the stimulus delivery computer (as brief DC shifts /
+# squarewave pulses). These pulses (often called "triggers") are used in this
+# dataset to mark experimental events: stimulus onset, stimulus type, and
+# participant response (button press). The individual STIM channels are
+# combined in a binary weighted sum onto a single channel, such that voltage
```

Comment:
I suppose the "binary weighted" bit is not necessary. You could just say that it is an integer and the integer value is unique for each stimulus